go113.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.13
+
+package errors
+
+import (
+	stderrors "errors"
+)
+
+func Is(err, target error) bool { return stderrors.Is(err, target) }
+
+func As(err error, target interface{}) bool { return stderrors.As(err, target) }
+
+func Unwrap(err error) error { return stderrors.Unwrap(err) }
```

Comment:
Maybe Unwrap could continue in normal package, because you can also use with a xerrors library? what do you think, @puellanivis, @Sherlock-Holo?

Reply:
`Unwrap` doesn't use any black magic (errors use `reflectlite` not `reflect`), it just uses type assert, we could implement it too